fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/utils/JcrPropertyStatementListener.java

```diff
@@ -50,33 +57,32 @@
 
     private final JcrRdfTools jcrRdfTools;
 
+    private final Session session;
+
     private final IdentifierConverter<Resource, FedoraResource> idTranslator;
 
     private final List<Exception> exceptions;
 
-    /**
-     * Construct a statement listener within the given session
-     *
-     * @param idTranslator the id translator
-     * @param session the session
-     */
-    public JcrPropertyStatementListener(final IdentifierConverter<Resource, FedoraResource> idTranslator,
-                                        final Session session) {
-        this(idTranslator, new JcrRdfTools(idTranslator, session));
-    }
+    private final Skolemizer skolemizer;
+
+    private final Service<Container> skolemAndHashCreator;
```

Comment:
I disagree. `containerService` is just a restatement of the type of the param. `skolemAndHashCreator` actually explains how the param is used. I'd like to hear from someone else.